packages/instrument-utils/src/storedInstrument/AccountInstrumentSelect/AccountInstrumentSelect.spec.tsx

```diff
@@ -240,6 +244,31 @@ describe('AccountInstrumentSelect', () => {
         );
     });
 
+    it('shows list of instruments when clicked and is an braintree ach instrument', () => {
```

Comment:
Don't forget to remove this one:
```suggestion
    it('shows list of instruments when clicked and is an ach instrument', () => {
```